### ISSUE TITLE
Can't reach the results on the paper 

### ISSUE THREAD

**Issue by ccnankai** [ZJULearning/pixel_link#47]
Hi,
@DelightRun 
I used my own data set for training, the test results are not good. I have trained 20,000 steps, and The loss function stops and does not reduced( stop at 0.6).
Can you give some Suggestions for improvement to get better results？Which parameters need to be changed？

![image](https://user-images.githubusercontent.com/33537489/43322920-b5456ea6-91e2-11e8-8858-8abe5c789eaa.png)

Thank you.
@comzyh

**Comment by greatgeekgrace** [comment]
Hi, How do you train that ~~I got a problem:
1. 刚开始时loss就很低0.0010左右，等到2000step时就成0.0000了
2. 训练的结果，只是框了红线部分，其他没有框
![image](https://user-images.githubusercontent.com/15572156/44194661-3a817200-a169-11e8-9e82-e2e3f92774bb.png)
多谢您了~~

**Comment by ccnankai** [comment]
@greatgeekgrace   我现在也遇到了同样问题，框的一点儿也不准。

**Comment by ccnankai** [comment]
@greatgeekgrace   我记得应该不用调什么，把样本标记好就可以了。

**Comment by MarStarck** [comment]
@greatgeekgrace 我训练到18 step的时候loss就是0了，蛋疼

**Comment by ccnankai** [comment]
@MarStarck   我训练损失函数都降不下来

**Comment by Simon-LLong** [comment]
> @MarStarck 我训练损失函数都降不下来

很难下降不知道为啥

**Comment by DelightRun** [comment]
@ccnankai @MarStarck @EvenIsLee 
Hi, all. Sorry for the late reply. 
Actually I and @comzyh are both not the original author of this project, please contact @dengdan.
